compiler/record-minmax-thread-safety-test/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+if(NOT ENABLE_TEST)
+    return()
+endif(NOT ENABLE_TEST)
+
```

Comment:
Please enable this test only in 64bit arch.

See https://github.com/Samsung/ONE/pull/10317 for details.

Reply:
Done, thank you!